Lobe_iOS/README.md

```diff
@@ -1,8 +1,6 @@
 # Files in iOS Bootstrap
 
-This project adheres to [MVVM architecture](https://www.raywenderlich.com/34-design-patterns-by-tutorials-mvvm). MVVM is a design pattern which organizes objects by model, view model, and view. Our app leverages MVVM as follows:
-
-![Code Diagram](https://github.com/lobe/iOS-bootstrap/raw/master/assets/codeDiagram.png)
```

Comment:
Any reason for removing this @RamonGilabert? I believe this schematic is really beneficial for helping developers understand the code snippets. There' a lot of moving parts here.

Reply:
Was mainly to reduce size of the download. We can revert this change and add the file back if you think it's beneficial—this is in the specific README, too, right?

Reply:
I should be able to reduce the size!